src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,12 @@
+.catalog {
+  padding: 50px 128px 72px;
+
+  &__grid {
+    display: grid;
+
+    grid-template-columns: repeat(4, 200px);
+    justify-content: center;
+    grid-row-gap: 46px;
+    grid-column-gap: 48px;
+  }
```

Comment:
What about this point from task description
<img width="544" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/b71c5adc-7289-4dcf-a13b-503ed5c5e5c1">